src/test/java/teammates/test/BaseTestCase.java

```diff
@@ -85,6 +99,48 @@ protected SqlDataBundle loadSqlDataBundle(String jsonFileName) {
         }
     }
 
+    protected Account getTypicalAccount() {
```

Comment:
Would it be okay to add a block comment explaining what these methods are for/how can these be used? So that in the future if we need to add more getTypicalX, the developer know they should put it here